sdk/nodejs/output.ts

```diff
@@ -175,7 +178,7 @@ This function may throw in a future version of @pulumi/pulumi.`;
             return message;
         };
 
-        this.apply = <U>(func: (t: T) => Input<U>) => {
+        this.apply = <U>(func: (t: T) => Input<U>, runWithUnknowns?: boolean) => {
```

Comment:
doc this at this point.  worth mentioning a few things:

1. this isn't part of the public API for .apply (afaik)
2. what the expected semantics are.  
3. mention a general reason why we want this.

Reply:
Done.